python-for-beginners/17 - JSON/read_json.py

```diff
@@ -16,8 +16,10 @@
 
 # According to the documentation for the analyze image function 
 # There are three optional parameters: language, details & visualFeatures
-parameters  = {'visualFeatures':'Description,Color',
-               'language':'en'}
+parameters  = {
```

Comment:
`parameters = {` 
This might related to indent(maybe some align), fixing or not is on your own.